waspc/src/Wasp/Analyzer/Parser/Lexer/Internal.hs

```diff
@@ -57,9 +57,9 @@ getInput = lstateInput <$> get
 updateInput :: Int -> Lexer ()
 updateInput !consumed = do
   (LexInput _ _ remaining) <- getInput
-  let newInput =
-        let (prevChar : remaining') = drop (consumed - 1) remaining
```

Comment:
Newer versions of GHC throw errors on pattern matches like this one ([already encoutered](https://github.com/wasp-lang/strong-path/pull/54#discussion_r2472330271) in Strong Path).

I used the same pattern/message we use in the rest of the project.